src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -8,60 +8,52 @@
 
 #include <ShellScalingApi.h>
 #include <mutex>
+#include <fileapi.h>
 
 #include <gdiplus.h>
 
 namespace ZoneWindowUtils
 {
-    const std::wstring& GetActiveZoneSetTmpPath()
+    const wchar_t ActiveZoneSetsTmpFileName[] = L"FancyZonesActiveZoneSets.json";

```

Comment:
I'm not sure if this is the right moment to change this, but I really don't see the point of these tmp file paths calculated in `ZoneWindowUtils`. They should be in `JSONHelpers` and not exposed to rest of `FancyZones`.

Reply:
IIRC, these were placed somewhere else initially, but at some point moved to ZoneWindow

Reply:
Could be, but currently I don't see why is that needed. Also by moving it into `FancyZones.cpp` certain `JsonHelpers` interface methods will not need it as an argument:
```
void ParseDeviceInfoFromTmpFile(std::wstring_view tmpFilePath);
bool ParseCustomZoneSetFromTmpFile(std::wstring_view tmpFilePath);
bool ParseDeletedCustomZoneSetsFromTmpFile(std::wstring_view tmpFilePath);
void SerializeDeviceInfoToTmpFile(const DeviceInfoJSON& deviceInfo, std::wstring_view tmpFilePath) const;
```

Reply:
Makes sense. I can do it as a follow-up